src/simsopt/field/mgrid.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+#import netCDF4 as nc
+from scipy.io import netcdf as nc
+import sys
+
+### File I/O ###
+# This class is prepared by Tony Qian (tqian@pppl.gov)
+# the original file included reading options for netCDF and binary
+# these have been removed and the code only writes netCDF
+#
+# updated 15 November 2021
+
+
+class MGRID():
+
+    def __init__(self, fname='temp', binary=False,\
+                 nr=51, nz=51, nphi=24, nfp=2,\
+                 rmin=0.20, rmax=0.40, zmin=-0.10, zmax=0.10,\
+                 nextcur=0):
+
+        self.nr = nr
+        self.nz = nz
+        self.nphi = nphi
+        self.nfp = nfp
+
+        self.rmin = rmin
+        self.rmax = rmax
+        self.zmin = zmin
+        self.zmax = zmax
+
+        self.n_ext_cur = 0
+        self.cur_labels = []
+
+        self.br_arr = [] 
+        self.bz_arr = [] 
+        self.bp_arr = [] 
+
+        print('Initialized mgrid file: (nr,nphi,nz,nfp) = ({}, {}, {}, {})'.format(nr, nphi, nz, nfp))
```

Comment:
You can use formatted string such as `f"Initializized mgrid file: nr = {nr}"`, instead of the `format` string method.

Reply:
Thanks for the tip!